libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -180,11 +257,20 @@ string IRGenerator::generate(Block const& _block)
 	return generator.code();
 }
 
-void IRGenerator::generateQueuedFunctions()
+set<FunctionDefinition const*> IRGenerator::generateQueuedFunctions()
```

Comment:
Does it make sense to store this information in the context instead?

Reply:
Ah this is also discussed here: https://github.com/ethereum/solidity/pull/10973/files#r578037647

Reply:
Yeah. In short, I think it would if not for the fact that the context gets reset during `generate()`.

I'll mark this one as resolved to keep discussion under the other comment.